pkgs/development/libraries/inchi/default.nix

```diff
@@ -8,10 +7,10 @@
 
 stdenv.mkDerivation {
   pname = "inchi";
-  version = "1.05";
+  version = "1.06";
   src = fetchurl {
-    url = "http://www.inchi-trust.org/download/105/INCHI-1-SRC.zip";
-    sha1 = "e3872a46d58cb321a98f4fd4b93a989fb6920b9c";
+    url = "http://www.inchi-trust.org/download/106/INCHI-1-SRC.zip";
```

Comment:
We should be able to generate that version number by replacing the dot with nothing.